src/main/java/Jarvis.java

```diff
@@ -0,0 +1,143 @@
+                    taskList.get(toMark).unmark();
+                    String markResponse = "Ok, I have marked this task as not done yet:\n ";
+                    System.out.println(markResponse + taskList.get(toMark).toString());
+                    continue;
+                }
+                //if userinput equals delete, check which task and delete
+                if (userInput.length() > 6 && userInput.substring(0, 6).equals("delete")) {
+                    int toDelete = Integer.parseInt(userInput.substring(7)) - 1;
+                    Task deleteTask = taskList.get(toDelete);
+                    taskList.remove(toDelete);
+                    String deleteResponse = "Noted. I have removed this task:\n ";
+                    System.out.println(deleteResponse + deleteTask.toString());
+                    continue;
+                }
+
+                //if userinput equals to do add new to do task to list
+                if (userInput.length() > 3 && userInput.substring(0, 4).equals("todo")) {
+                    String description = userInput.substring(4);
+                    if (description.equals("")) {
+                        throw new JarvisException("The description of a todo cannot be empty");
+                    }
+                    taskList.add(new ToDo(description));
+                    System.out.println("Got it. I've added this task:\n " + taskList.get(Task.count - 1)
+                            + "\nNow you have " + (Task.count) + " tasks in the list.");
+                    continue;
+                }
+
+                //if userinput equals deadline add new deadline task to list
+                if (userInput.length() > 8 && userInput.substring(0, 8).equals("deadline")) {
+                    int divisor = userInput.indexOf("/by");
+                    String description = userInput.substring(9, divisor - 1);
+                    String date = userInput.substring(divisor + 4);
+                    taskList.add(new Deadline(description, date));
+                    System.out.println("Got it. I've added this task:\n " + taskList.get(Task.count - 1)
+                            + "\nNow you have " + (Task.count) + " tasks in the list.");
+                    continue;
+                }
+                //if userinput equals event add new event task to list
+                if (userInput.length() > 7 && userInput.substring(0, 5).equals("event")) {
+                    int divisor = userInput.indexOf("/at");
+                    String description = userInput.substring(6, divisor - 1);
+                    String date = userInput.substring(divisor + 4);
+                    taskList.add(new Event(description, date));
+                    System.out.println("Got it. I've added this task:\n " + taskList.get(Task.count - 1)
+                            + "\nNow you have " + (Task.count) + " tasks in the list.");
+                    continue;
+                }
```

Comment:
Perhaps changing this to a switch statement would be better, it'd improve code readability.